uber/server.py

```diff
@@ -11,6 +11,25 @@ def _add_email():
 cherrypy.tools.add_email_to_error_page = cherrypy.Tool('after_error_response', _add_email)
 
 
+def on_cherrypy_before_error_response(debug=False):
```

Comment:
``on_cherrypy_before_error_response`` is not a great name for a tool, since it doesn't actually say what the tool does.  I would suggest something like ``log_error_contexts`` or something, since that's actually descriptive about what this is doing.